scrabble-score.json

```diff
@@ -0,0 +1,59 @@
+{
+  "cases": [
+    {
+      "description": "lowercase letter",
+      "input": "a",
+      "expected": 1
+    },
+    {
+      "description": "uppercase letter",
+      "input": "A",
+      "expected": 1
+    },
+    {
+      "description": "valuable letter",
+      "input": "f",
+      "expected": 4
+    },
+    {
+      "description": "short word",
+      "input": "at",
+      "expected": 2
+    },
+    {
+      "description": "short, valuable word",
+      "input": "zoo",
+      "expected": 12
+    },
+    {
+      "description": "medium word",
+      "input": "street",
+      "expected": 6
+    },
+    {
+      "description": "medium, valuable word",
+      "input": "quirky",
+      "expected": 22
+    },
+    {
+      "description": "long, mixed-case word",
+      "input": "OxyphenButazone",
+      "expected": 41
+    },
+    {
+      "description": "english-like word",
+      "input": "pinata",
+      "expected": 8
+    },
+    {
+      "description": "non-english letter is not scored",
+      "input": "piñata",
+      "expected": 7
```

Comment:
Expected, 7, so it is scored, the title is not truthful.  The ñ'' is not scored.